### PR TITLE
tests: subsys: zbus: do not use verbose logging

### LINKED ISSUE #63409: subsys: zbus: failing tests on qemu_riscv64_smp and qemu_riscv32_smp
**Describe the bug**
The test `tests/subsys/zbus/integration/message_bus.zbus.module_interaction_no_error` fails on `qemu_riscv64_smp` and `qemu_riscv32_smp`. 

Please also mention any information which could help others to understand
the problem you're facing:
 - What target platform are you using? `qemu_riscv64_smp` and `qemu_riscv32_smp`
 - What have you tried to diagnose or workaround this issue? step-debugged, did a bisect
 - Is this a regression? yes, at 0b0aa435afc8b0102b437ae145203daada6c616b

**To Reproduce**
1. `git bisect start`
2. `git bisect bad HEAD`
3. `git bisect good v3.4.0`
4. `git bisect run /tmp/test.sh`

See below for `/tmp/test.sh` contents

**Expected behavior**
Tests should pass.

**Impact**
Breaks CI, on multiple unrelated PR's. So a showstopper

**Logs and console output**
```shell
west build -p auto -b qemu_riscv64_smp -t run tests/subsys/zbus/integration/
...
*** Booting Zephyr OS build v3.5.0-rc1-32-g4435646c7b2b ***
Running TESTSUITE integration
===================================================================
START - test_basic
D: Notifing start_measurement_chan's observers. Starting VDED:
D:  0 -> 
I:  *** LISTENER activated for channel start_measurement_chan ***

D: [Peripheral] starting measurement
D:  1 -> 
D: [Peripheral] sending sensor data
D: Notifing sensor_data_chan's observers. Starting VDED:
D:  0 -> 
D: Sensor {a = 1, b = 1}. Sending pkt {total=2}
D: Notifing net_pkt_chan's observers. Starting VDED:
D:  0 -> 
D: [Net] Total 2
D: Notifing net_log_chan's observers. Starting VDED:
D:  0 -> 
D: [Net log]: count_net = 1, pkt.total = 2
D: Notifing start_measurement_chan's observers. Starting VDED:
D:  0 -> 
I:  *** LISTENER activated for channel start_measurement_chan ***

D: [Peripheral] starting measurement
D:  1 -> 
D: [Peripheral] sending sensor data
D: Notifing sensor_data_chan's observers. Starting VDED:
D:  0 -> 
D: Sensor {a = 1, b = 1}. Sending pkt {total=2}
D: Notifing net_pkt_chan's observers. Starting VDED:
D:  0 -> 
D: [Net] Total 2
D: Notifing net_log_chan's observers. Starting VDED:
D:  0 -> 
D: [Net log]: count_net = 2, pkt.total = 2
D: Notifing start_measurement_chan's observers. Starting VDED:
D:  0 -> 
I:  *** LISTENER activated for channel start_measurement_chan ***

D: [Peripheral] starting measurement
D:  1 -> 
D: [Peripheral] sending sensor data
D: Notifing sensor_data_chan's observers. Starting VDED:
D:  0 -> 
D: Sensor {a = 1, b = 1}. Sending pkt {total=2}
D: Notifing net_pkt_chan's observers. Starting VDED:
D:  0 -> 
D: [Net] Total 2
D: Notifing net_log_chan's observers. Starting VDED:
D:  0 -> 
D: [Net log]: count_net = 3, pkt.total = 2

    Assertion failed at WEST_TOPDIR/zephyr/tests/subsys/zbus/integration/src/main.c:215: integration_test_basic: (pkt.total not equal to 6)
result was 2
 FAIL - test_basic in 1.633 seconds
===================================================================
START - test_channel_set_enable
D: Notifing start_measurement_chan's observers. Starting VDED:
D: Notifing start_measurement_chan's observers. Starting VDED:
D:  0 -> 
D: [Peripheral] starting measurement
D: [Peripheral] sending sensor data
D: Notifing sensor_data_chan's observers. Starting VDED:
D:  0 -> 
D: Sensor {a = 1, b = 1}. Sending pkt {total=2}
D: Notifing net_pkt_chan's observers. Starting VDED:
D:  0 -> 
D: [Net] Total 2
D: Notifing net_log_chan's observers. Starting VDED:
D:  0 -> 
D: [Net log]: count_net = 1, pkt.total = 2
D: Notifing start_measurement_chan's observers. Starting VDED:
I:  *** LISTENER activated for channel start_measurement_chan ***

D:  0 -> 
D: Notifing start_measurement_chan's observers. Starting VDED:
D:  0 -> 
I:  *** LISTENER activated for channel start_measurement_chan ***

D: [Peripheral] starting measurement
D:  1 -> 
D: [Peripheral] sending sensor data
D: Notifing sensor_data_chan's observers. Starting VDED:
D:  0 -> 
D: Sensor {a = 1, b = 1}. Sending pkt {total=2}
D: Notifing net_pkt_chan's observers. Starting VDED:
D:  0 -> 
D: [Net] Total 2
D: Notifing net_log_chan's observers. Starting VDED:
D:  0 -> 
D: [Net log]: count_net = 2, pkt.total = 2

    Assertion failed at WEST_TOPDIR/zephyr/tests/subsys/zbus/integration/src/main.c:276: integration_test_channel_set_enable: (pkt.total not equal to 4)
result was 2
 FAIL - test_channel_set_enable in 1.836 seconds
===================================================================
START - test_event_dispatcher_mutex_timeout
D: Notifing busy_chan's observers. Starting VDED:
 PASS - test_event_dispatcher_mutex_timeout in 3.015 seconds
===================================================================
START - test_event_dispatcher_queue_timeout
D: Notifing start_measurement_chan's observers. Starting VDED:
D:  0 -> 
I:  *** LISTENER activated for channel start_measurement_chan ***

D: [Peripheral] starting measurement
D:  1 -> 
D: [Peripheral] sending sensor data
D: Notifing sensor_data_chan's observers. Starting VDED:
D: Notifing start_measurement_chan's observers. Starting VDED:
D:  0 -> 
I:  *** LISTENER activated for channel start_measurement_chan ***

D:  1 -> 
D: Notifing start_measurement_chan's observers. Starting VDED:
E: could not deliver notification to observer . Error code -11
D:  0 -> 
I:  *** LISTENER activated for channel start_measurement_chan ***

D:  1 -> 
D: [Peripheral] starting measurement
D: [Peripheral] sending sensor data
D: Notifing sensor_data_chan's observers. Starting VDED:
 PASS - test_event_dispatcher_queue_timeout in 1.668 seconds
===================================================================
TESTSUITE integration failed.

------ TESTSUITE SUMMARY START ------

SUITE FAIL -  50.00% [integration]: pass = 2, fail = 2, skip = 0, total = 4 duration = 8.152 seconds
 - FAIL - [integration.test_basic] duration = 1.633 seconds
 - FAIL - [integration.test_channel_set_enable] duration = 1.836 seconds
 - PASS - [integration.test_event_dispatcher_mutex_timeout] duration = 3.015 seconds
 - PASS - [integration.test_event_dispatcher_queue_timeout] duration = 1.668 seconds

------ TESTSUITE SUMMARY END ------

===================================================================
PROJECT EXECUTION FAILED
```

**Environment (please complete the following information):**
 - OS: (e.g. Linux, MacOS, Windows): Linux
 - Toolchain (e.g Zephyr SDK, ...): Zephyr SDK v0.16.1
 - Commit SHA or Version used: ea4a463cc00d37058cabc84e22c030eafc10a19d

**Additional context**

### PR DESCRIPTION
**Pull request by cfriedt**
This change fixes a regression on `qemu_riscv64_smp` and `qemu_riscv32_smp` that arose because debug logging was enabled. My guess is that subtle races exist and that debug logging exacerbates the conditions for those races.

Fixes #63409

Testing Done:
```shell
twister -i -p qemu_riscv64_smp -p qemu_riscv32_smp -T tests/subsys/zbus/integration
...
INFO    - Using Ninja..
INFO    - Zephyr version: v3.5.0-rc1-32-gc74a7ef2f6b4
INFO    - Using 'zephyr' toolchain.
INFO    - Building initial testsuite list...
INFO    - Writing JSON report /Users/cfriedt/zephyrproject/zephyr/twister-out/testplan.json
INFO    - Adding tasks to the queue...
INFO    - Added initial list of jobs to queue
INFO    - Total complete:    2/   2  100%  skipped:    0, failed:    0, error:    0
INFO    - 1 test scenarios (2 test instances) selected, 0 configurations skipped (0 by static filter, 0 at runtime).
INFO    - 2 of 2 test configurations passed (100.00%), 0 failed, 0 errored, 0 skipped with 0 warnings in 29.08 seconds
INFO    - In total 8 test cases were executed, 0 skipped on 2 out of total 626 platforms (0.32%)
INFO    - 2 test configurations executed on platforms, 0 test configurations were only built.
INFO    - Saving reports...
INFO    - Writing JSON report /Users/cfriedt/zephyrproject/zephyr/twister-out/twister.json
INFO    - Writing xunit report /Users/cfriedt/zephyrproject/zephyr/twister-out/twister.xml...
INFO    - Writing xunit report /Users/cfriedt/zephyrproject/zephyr/twister-out/twister_report.xml...
INFO    - Run completed
```

### CODE REVIEW

**Review by fabiobaltieri — APPROVED** [review]

**Review by jgl-meta — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Comment by jhedberg** [comment]
Not sure what the official process is with hotfixes and waiting for the maintainer’s approval, but this change seems trivial enough that I’ll go ahead and merge it. @rodrigopex hopefully that’s ok with you.

**Pull request merged**
